src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -561,11 +562,15 @@
 				"\tset the cost of enchant offer 1 to 10 levels")
 			.since("2.5");
 		Skript.registerEvent("Enchant", SimpleEvent.class, EnchantItemEvent.class, "[item] enchant")
-		.description("Called when a player successfully enchants an item.",
-			" To get the enchanted item, see the <a href='expressions.html#ExprEnchantEventsEnchantItem'>enchant item expression</a>")
-		.examples("on enchant:",
-			"\tif the clicked button is enchantment option 1:",
-			"\t\tset the applied enchantments to sharpness 10 and unbreaking 10")
-		.since("2.5");
+			.description("Called when a player successfully enchants an item.",
+				" To get the enchanted item, see the <a href='expressions.html#ExprEnchantEventsEnchantItem'>enchant item expression</a>")
+			.examples("on enchant:",
+				"\tif the clicked button is enchantment option 1:",
+				"\t\tset the applied enchantments to sharpness 10 and unbreaking 10")
+			.since("2.5");
```

Comment:
As stated previously in another PR, line changes should only be done for things relevant to the theme of the PR.